source/server/http/admin.cc

```diff
@@ -426,6 +428,25 @@ Http::Code AdminImpl::handlerConfigDump(absl::string_view, Http::HeaderMap& resp
   return Http::Code::OK;
 }
 
+Http::Code AdminImpl::handlerContention(absl::string_view, Http::HeaderMap& response_headers,
+                                        Buffer::Instance& response, AdminStream&) {
+
+  if (server_.options().mutexTracingEnabled()) {
+    response_headers.insertContentType().value().setReference(
+        Http::Headers::get().ContentTypeValues.Json);
+
+    envoy::admin::v2alpha::MutexStats mutex_stats;
+    mutex_stats.set_num_contentions(MutexTracer::getOrCreateTracer()->numContentions());
+    mutex_stats.set_current_wait_cycles(MutexTracer::getOrCreateTracer()->currentWaitCycles());
+    mutex_stats.set_lifetime_wait_cycles(MutexTracer::getOrCreateTracer()->lifetimeWaitCycles());
+    response.add(MessageUtil::getJsonStringFromMessage(mutex_stats, true, true));
```

Comment:
set content-type to json properly here and in handlerMemory()

Reply:
Is that not what 
```
   435     response_headers.insertContentType().value().setReference(
   436         Http::Headers::get().ContentTypeValues.Json);
```
 does?

Reply:
errrr, yes; never mind.